src/BmiForm.jsx

```diff
@@ -31,7 +36,7 @@ const BmiForm = ({ change }) => {
           <input
             id="weight"
             name="weight"
-            type="tel"
+            type="number"
```

Comment:
In the case of `type="number"`  `maxLength="3"` will not work. That's why using "tel".